routes/promotionRouter.spec.js

```diff
@@ -1,89 +1,72 @@
-const supertest = require('supertest');
-const router = require('./promotionRouter.js');
+const supertest = require("supertest");
+const router = require("./promotionRouter.js");
 
-describe('promotion router', () => {
```

Comment:
hmm. you changed the formatting. are you using a different code formatter?

Reply:
I haven't consciously done any formatting changes but I do have Prettier on VS code